src/js/constants/BlackListServiceConfig.js

```diff
@@ -0,0 +1,16 @@
+let BlackListServiceConfig = [
+  'uris',
+  'ports',
+  'version',
+  'versionInfo',
+  'deployments',
+  'tasks',
+  'taskStats',
+  'tasksHealthy',
+  'tasksRunning',
+  'tasksStaged',
+  'tasksUnhealthy',
+  '_itemData'
```

Comment:
Yes, it's necessary. Without it we'll be passing the `_itemData` to the JSON editor:
![](https://cl.ly/3Y0O3f3J0H40/Screen%20Shot%202016-08-03%20at%2011.15.19%20AM.png)